argus/action_manager/windows.py

```diff
@@ -610,6 +610,7 @@ def prepare_config(self, cbinit_conf, cbinit_unattend_conf):
             cbinit_conf, cbinit_unattend_conf)
```

Comment:
replace commit message with: Skip user_data_save_path on Nano and in the description say that takeowner.exe does not exist on nano